From 6ff1cd1e487ddf545337b88da3f1f5ca69a2f958 Mon Sep 17 00:00:00 2001
From: Tim Kientzle <kientzle@acm.org>
Date: Mon, 6 May 2024 20:46:26 -0700
Subject: [PATCH] Define INT_MAX via `#include <limits.h>` (#2170)

#2110 added usages of INT_MAX here without adding the necessary header.

Resolves #2162
---
 tar/bsdtar.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/tar/bsdtar.c b/tar/bsdtar.c
index 42baab286..f0e71afd1 100644
--- a/tar/bsdtar.c
+++ b/tar/bsdtar.c
@@ -43,6 +43,9 @@
 #ifdef HAVE_LANGINFO_H
 #include <langinfo.h>
 #endif
+#ifdef HAVE_LIMITS_H
+#include <limits.h>
+#endif
 #ifdef HAVE_LOCALE_H
 #include <locale.h>
 #endif
